src/components/Header.jsx

```diff
@@ -10,7 +10,15 @@ function Header() {
         <Link className="nav-link" to={"/"}>
           Home
         </Link>
-        <Link className="nav-link">Topics</Link>
+        <Link to={"/topics/coding"} className="nav-link">
```

Comment:
Not an issue at the moment, but would be better if you used your api route to get the current topics. means if the database changes/more topics get added, the website is upto date